src/main/resources/db/changelog/changes/03-create-roles-table.yaml

```
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-roles-table
+      author: dmytro

```

```suggestion
      author: dmytro_potapov
```
it's better to provide full name as author of changeSet